keras/backend.py

```diff
@@ -5484,7 +5484,7 @@ def _get_logits(output, from_logits, op_type, fn_name):
 @keras_export("keras.backend.categorical_crossentropy")
 @tf.__internal__.dispatch.add_dispatch_support
 @doc_controls.do_not_generate_docs
-def categorical_crossentropy(target, output, from_logits=False, axis=-1):
+def categorical_crossentropy(target, output, from_logits=False, forced_rescale=True, axis=-1):
```

Comment:
with `from_logits` set to `False`, the output is already normalized. So this additional parameter is unnecessary